<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateCertificateAssistant</class>
 <widget class="QWizard" name="CreateCertificateAssistant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Create Certificate</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::NStyles</enum>
  </property>
  <property name="options">
   <set>QWizard::CancelButtonOnLeft|QWizard::IndependentPages|QWizard::NoBackButtonOnStartPage|QWizard::NoDefaultButton</set>
  </property>
  <widget class="gui::assistant::page::NameWidgetPage" name="page_namefields">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="title">
    <string>Subject &amp; Issuer</string>
   </property>
  </widget>
  <widget class="QWizardPage" name="page_keyPair">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Algorithm</string>
   </property>
   <layout class="QVBoxLayout" name="layout_page_keyPair">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="gui::widget::KeyPairWidget" name="keyPairWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="page_validity">
   <property name="title">
    <string>Validity</string>
   </property>
   <layout class="QVBoxLayout" name="layout_page_validity">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QFormLayout" name="validity_layout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::FieldsStayAtSizeHint</enum>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="validityperiod_label">
        <property name="text">
         <string>Validity (Period in Days)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="validityperiod_field">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>16777215</number>
        </property>
        <property name="value">
         <number>365</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="validtiyperiod_from_label">
        <property name="text">
         <string>Valid from</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="validityperiod_from_field">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="validityperiod_until_label">
        <property name="text">
         <string>Valid until</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="validityperiod_until_field">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="page_key_usage">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="gui::widget::KeyUsageWidget" name="keyUsageWidget" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>gui::widget::KeyPairWidget</class>
   <extends>QWidget</extends>
   <header location="global">src/gui/widget/KeyPairWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>gui::assistant::page::NameWidgetPage</class>
   <extends>QWizardPage</extends>
   <header location="global">src/gui/assistant/page/NameWidgetPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>gui::widget::KeyUsageWidget</class>
   <extends>QWidget</extends>
   <header location="global">src/gui/widget/KeyUsageWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
